doc/installation.xml

```diff
@@ -225,6 +225,45 @@ home-manager.useUserPackages = true;
     become the default value in the future.
    </para>
   </note>
+
+  <note>
+   <para>
+    By default, Home Manager modules use a private <literal>pkgs</literal> instance
+    that is configured via the <option>home-manager.users.&lt;name?&gt;.nixpkgs</option> options.
+    To instead use the global <literal>pkgs</literal> that is configured via
+    the <option>nixpkgs</option> options, set
+   </para>
+<programlisting language="nix">
+home-manager.useGlobalPkgs = true;
+</programlisting>
+  <para>
+   This saves an extra Nixpkgs evaluation, adds consistency, and removes the
+   dependency on <envar>NIX_PATH</envar>, which is otherwise used for
+   importing the pkgs.
+   This option may become the default value in the future.
+   </para>
+  </note>
+
+  <para>
```

Comment:
I would suggest removing this commit since it is orthogonal to the general PR and I think does not fit in the installation section. Maybe could go in a "tips and tricks" section (in a separate PR) or even on the [wiki](https://nixos.wiki/wiki/Home_Manager)?

Reply:
I've removed it. Where would you add that tips and tricks section?

Reply:
I guess there should be a new chapter called something like "NixOS Integration", with a section "Tips and Tricks" where this would go into a subsection.

I think there you could also have more space to write text surrounding this use pattern since it is dealing with more advanced concepts such as defining your own options and using a function like `mkAliasDefinitions`.